### ISSUE TITLE
Play from local playlist

### ISSUE THREAD

**Issue by soimort** [soimort/you-get#325]
might not be the most desired feature though, it's nice to have

``` console
$ you-get -p vlc favorite_pl_videos.txt
```

`favorite_pl_videos.txt:`

```
https://www.youtube.com/watch?v=vy5C-mlUQ1w
https://www.youtube.com/watch?v=7NPBrWDzO2A
https://www.youtube.com/watch?v=3q8xYFDYLeI
https://www.youtube.com/watch?v=b0EF0VTs9Dc
https://www.youtube.com/watch?v=ZKmodCApZwk
https://www.youtube.com/watch?v=O1t4xJzrOng
```

p.s.  I'm too busy at the moment to contribute anything to this project.

**Comment by rosynirvana** [comment]
#1736
This can be closed.

**Comment by cnbeining** [comment]
@rosynirvana FYI you can put something like ```fix #325 ```  in your comment log and that can automatically close the issue(s) referred.
